distributed/publish.py

```diff
@@ -116,3 +116,7 @@ def __len__(self):
                 "please use 'len(await client.list_datasets())' instead"
             )
         return len(self._client.list_datasets())
+
+    def get(self, key, default=None):
```

Comment:
Why is this needed?

Reply:
Yes, you are totally correct. It will pick up the default argument anyways. Thanks for the hint!